nextjs-13/src/app/page.test.tsx

```diff
@@ -22,92 +20,66 @@ describe('Home', () => {
     requestManager.mockResolvedValue({ records: [] });
   });
 
-  describe('Session status is "unauthenticated', () => {
-    it('redirects to home page', async () => {
-      useSession.mockReturnValue({ status: 'unauthenticated' });
+  it('renders title', async () => {
+    render(<Home />);
 
-      render(<Home />);
-
-      await waitFor(() => {
-        expect(redirect).toHaveBeenCalledWith('/auth/sign-in');
-      });
+    await waitFor(() => {
+      expect(screen.getByText('Your Newsletters')).toBeInTheDocument();
     });
   });
 
-  describe('Session status is "authenticated', () => {
+  describe('giving fetching data', () => {
```

Comment:
I missed this incorrect phrasing:

```suggestion
  describe('given fetching data', () => {
```

Reply:
Fixed [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/commits/108cc7cf1e36665071c8d25ff89f06df610ed39c#diff-4b9ae30a4e34a98e0e0d12af3e4872e7ac9865c68f016c1e6667d4336ce08ce1R31-R51)